### ISSUE TITLE
Search for articles with status=DRAFT points to published url

### ISSUE THREAD

**Issue by atraining** [vitorfs/bootcamp#95]
I usee the searchbox on the `home` url to find a article which has been saved as draft. I found the article, however when I click on it the url points to `articles/<title>/` and not to `articles/edit/2/` as drafts are not available via public url where only login is required.

![image](https://user-images.githubusercontent.com/2879188/28381869-830b4876-6cbc-11e7-80b6-69e726524f39.png)

**Comment by sebastian-code** [comment]
Hi @atraining I looked into it, and your totally right. Here the thing will be about functionality and how to solve it. Initially the scope was than the search view should bring the public content, which a draft is not. So, I have to think if I want to include the drafts into the search results, or to exclude them and allow it's search just through the drafts menu inside the articles app.

Actually for me, the right solution is the second option, but I'm open to hear other positions if you have some ideas.

**Comment by atraining** [comment]
Hi @sebastian-code, thanks for your response. In fact, it's a design pattern question. Despite the fact that users might want to search for articles and do not care about the state of the article - atm I would just filter for published articles.

```
# https://github.com/vitorfs/bootcamp/blob/master/bootcamp/search/views.py#L30
        results['questions'] = Question.objects.filter(status='PUBLISHED').filter(
            Q(title__icontains=querystring) | Q(description__icontains=querystring))
```

**Comment by sebastian-code** [comment]
Solved with fd66f3b2cafe44567c3efd7e8f3271fa5da0dcb9
